slogans.json

```diff
@@ -1,9 +1,9 @@
 {
   "2020-07-19": { "slogan": "just screw it" },
   "2020-07-20": { "slogan": "just blew it" },
-  "2020-07-21": { "slogan": "up for grabs", "via": "@your-twitter-handle" },
+  "2020-07-21": { "slogan": "just ship it", "via": "@_kamlesh_" },
   "2020-07-22": { "slogan": "just knew it" },
-  "2020-07-23": { "slogan": "up for grabs", "via": "@your-twitter-handle" },
+  "2020-07-23": { "slogan": "just keep it simple sweet", "via": "@_kamlesh_" },
```

Comment:
doesn't, have it in the end :P

updated the format to signal this : https://github.com/siddharthkp/just-do-it-pls/blob/main/slogans.json